src/main/java/reposense/util/StringsUtil.java

```diff
@@ -31,10 +31,33 @@ public static String replaceSpecialSymbols(String regexString, String replacemen
         return SPECIAL_SYMBOLS.matcher(regexString).replaceAll(replacementCharacter);
     }
 
-    public static String addQuote(String original) {
+    /**
+     * Adds quotes to the string.
+     */
+    public static String addQuotes(String original) {
         return "\"" + original + "\"";
     }
 
+    /**
+     * Adds the appropriate quotation marks for a file path depending on the OS.
+     */
+    public static String addQuotesForFilePath(String filePath) {
```

Comment:
It might be possible that developers forget to use these if they weren't sure how or when to use them. Do you think it would be a good idea now to create a CommandBuilder to take care of these things.

This could be tricky though, given that the construction of commands can be non-trivial so this might be out of scope for this PR.

Reply:
Interesting. I actually voiced it [here](https://github.com/reposense/RepoSense/pull/1562#issuecomment-877820941) and you can read the response from dcshzj but besides adding paths, most other argument types are quite tailored to the specific git command. So this is a specific standard for file paths only.

Reply:
Thanks for the link! It's interesting seeing similar discussions and considerations springing up independently at different points in this project :)